packages/now-build-utils/package.json

```
@@ -16,6 +16,10 @@
     "test-integration-once": "jest --env node --verbose --runInBand test/integration.test.js",
     "prepublishOnly": "./build.sh"
   },
+  "dependencies": {
+    "@now/frameworks": "^0.0.7",
+    "@now/routing-utils": "^1.5.2-canary.2"
```

```suggestion
    "@now/routing-utils": "1.5.2-canary.3"
```

Let's use the exact version since we publish lerna with `--exact`